### PR TITLE
Docs 5451 db connector template implementation

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/database/database-connector-config-topics.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-config-topics.adoc:11]: Looks like there is no configuration information for DB. If this is the case, then this file needs to be removed from the PR and not added to the latest version.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: change: will use to uses.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Not sure why this is a note? Could it just be part of a step instead?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: What does Kim think about 'Tips'?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: This is not in backticks - may be an extra space?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Again about 'Tips'. Also, there are a lot of 'wills' that can be rewritten to present tense.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc:228]: I don't know DBs so I assume these sentences make sense?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc:244]: Given your understanding of DBs, does this make sense to be in here? That is, will the Mule 4 user get tripped up if they don't have this info?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Is this a straggler from the stuff you got from Dev? Needs to be reworded.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Again, is this question useful here? I think it attempts to answer the question with an example, but I was also looking for a definition as an answer. Perhaps reword?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Rewrite the 'wills'

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc:259]: Is this important to how someone would have set up their DBs in Mule 3 vs. Mule 4?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: REmove the xref here.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Why is this a note? Couldn't it just be part of the explanation of 'fetchSize'? My def of a note is if the user will run into something that can trip them up. Not just a "nice to know".

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Removed xref.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Check your heading levels throughout. This looks like it's at the same level as the Example intro. I think you added this info to this section. Did you decide that it needed to be under one header as opposed to making these examples part of a longer list?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: The main title is Example. This title should not use the word, Example.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc:317]: I thought you were going to rewrite this?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: REmove xref

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Again, retitle without the word, Example.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: As previous, Example.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: formatting?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Tip or just part of the instructions above?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: You list some of these as Mule 4. Is there a reason for that?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: REmove xref.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Mule 4?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc:575]: Hmm, seems like this is good info to have *before* I have read all this. Is there a better spot to discuss bulk operations?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Check your heading levels throughout.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: Is this image important?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-examples.adoc]: formatting for <try>?

File: modules/ROOT/pages/database/database-connector-get-started.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-get-started.adoc:13]: Hmm, I've never heard of the Mule palette before. Can you confirm that this is correct?

File: modules/ROOT/pages/database/database-connector-get-started.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-get-started.adoc]: This should go after About Connectors.

File: modules/ROOT/pages/database/database-connector-get-started.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-get-started.adoc:49]: Do you have additional configurations? I didn't see any. If not, you'll need to change this.

File: modules/ROOT/pages/database/database-connector-studio.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-studio.adoc]: Where are these? Why aren't they here? This isn't really useful if it doesn't point me to where the Global Properties are.

File: modules/ROOT/pages/database/database-connector-xml-maven.adoc
Comment (mcherem) [inline modules/ROOT/pages/database/database-connector-xml-maven.adoc]: Please check the formatting from here on out. It was wrong in the templates. It has been fixed but doesn't help you. all filenames need to be in backticks. For example, `pom.xml`.

**Review by mcherem — CHANGES_REQUESTED** [review]
Peer reviewed, a few fixes need to be made before approval.

File: modules/ROOT/pages/database/database-connector-config-topics.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-config-topics.adoc:11]: Will remove.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Accepted.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: There aren't really steps here. I could just remove the note designation, but it was probably made into a note to call it out.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Unnecessary. Removed tip designation.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Checked the style guide and removed the note designation.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Extra space. Removed.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Searched and removed all wills and tips.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc:228]: Makes sense to me.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc:244]: This is info they need if they previously used Mule 3. This influences how their queries are written with respect to select.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Straggler. Rewording.

Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Dynamic queries are necessary in the above example and cannot be treated like Input parameters; Input parameters can only be applied to parameters in a `WHERE` clause. To modify any other part of the query, you should use DataWeave’s interpolation operator.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed question and added "For example," before "suppose".

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed all wills.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc:259]: Yes, if they are transitioning.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed the note designation. This should probably still be there but doesn't need to be called out.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: I have not touched the heading levels. This particular topic had many h2 examples underneath it. Should each example topic be h2? Is h3 supported?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc:317]: This is the rewrite. Removed xref in the meantime.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed formatting. Was making a reference to the table made in the Insert section. Named that since I am not doing an in-topic reference.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed tip designation.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Not sure. I would assume it is all Mule 4?

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Removed.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Can make h1 to h2 and h2 to h3? Will ping you.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: No. Removed.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc]: Added `<try>`

File: modules/ROOT/pages/database/database-connector-get-started.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-get-started.adoc]: Moved.

File: modules/ROOT/pages/database/database-connector-get-started.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-get-started.adoc:49]: No, I do not.

File: modules/ROOT/pages/database/database-connector-get-started.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-get-started.adoc:13]: Asked Sebastian.

File: modules/ROOT/pages/database/database-connector-get-started.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-get-started.adoc:49]: Removed the link from the list.

File: modules/ROOT/pages/database/database-connector-get-started.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-get-started.adoc:13]: Yes, it is still a thing and is correct.

File: modules/ROOT/pages/database/database-connector-studio.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-studio.adoc]: They're a generic topic. Added a link.

File: modules/ROOT/pages/database/database-connector-xml-maven.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-xml-maven.adoc]: Added backticks.

File: modules/ROOT/pages/database/database-connector-examples.adoc
Comment (sarathecoubian) [reply modules/ROOT/pages/database/database-connector-examples.adoc:575]: Moved a short desc about bulk operations to the top but left the example of bad operations intact.

**Review by mcherem — APPROVED** [review]

**Pull request merged**
